recipes/paho-mqtt-c/all/conanfile.py

```diff
@@ -111,9 +115,14 @@ def package_info(self):
                 self.cpp_info.components["_paho-mqtt-c"].system_libs.extend(["c"])
             else:
                 self.cpp_info.components["_paho-mqtt-c"].system_libs.extend(["c", "pthread"])
+
         if self.options.ssl:
             self.cpp_info.components["_paho-mqtt-c"].requires = ["openssl::openssl"]
 
+    @property
```

Comment:
I think it would help reading if all the properties would be defined on the top, before used
but that might be personal taste

Reply:
Or just after their first usage. At least, close to their usage.